src/app/login/githublogin.tsx

```diff
@@ -32,11 +47,21 @@ const GithubLogin: React.FC = () => {
                 icon={<GithubIcon />}
                 iconPosition="left"
                 size="lg"
-                style={{ backgroundColor: 'black' }}
+                style={{ backgroundColor: 'black', marginRight: '20px' }}
```

Comment:
Mind flipping the order of these buttons so that "Sign in with GitHub" is on the right instead?

For left-to-right readers, placing the one-time/required button on the left will make it likelier to be noticed and clicked on first. Hopefully in the future we won't need two buttons, but reordering would work as a quick improvement for now.